packages/storybook/stories/va-radio.stories.jsx

```diff
@@ -13,19 +13,30 @@ export default {
   subcomponents: componentStructure(radioItem),
 };
 
-const Template = args => {
+const Template = ({
+  'enable-analytics': enableAnalytics,
+  error,
+  label,
+  required,
+}) => {
   return (
-    <va-radio {...args}>
+    <va-radio
+      enable-analytics={enableAnalytics}
+      error={error}
+      label={label}
+      required={required}
+    >
       <va-radio-option label="Option one" name="one" value="1" />
       <va-radio-option label="Option two" name="two" value="2" />
     </va-radio>
   );
 };
 
 const defaultArgs = {
-  label: 'This is a label',
-  required: false,
-  error: null,
+  'enable-analytics': false,
```

Comment:
This gives us an extra arg:

![image](https://user-images.githubusercontent.com/2008881/142943464-834a57d4-7338-4237-bda0-0cd47377735f.png)

Reply:
Good catch. The latest changes use the components prop attribute instead of prop name.